src/api/index.ts

```
@@ -1,18 +1,63 @@
+      signal: AbortSignal.timeout
+        ? AbortSignal.timeout(ABORT_MS)
+        : abortPolyfill(ABORT_MS),
+    });
+
+    return data as Awaited<ReturnType<typeof fetch>>;
+  } catch (err: any) {
+    if ((err as Error).name === 'TimeoutError')
+      throw new ServerNotRespondingError();
+    else {
+      if (!error || !(error && error[`${(err as Response).status}`]))
+        throw new Error(`알 수 없는 오류가 발생했습니다 ${err.name}`);
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)